.circleci/scripts/binary_linux_test.sh

```diff
@@ -19,10 +19,22 @@ fi
 # Install the package
 # These network calls should not have 'retry's because they are installing
 # locally and aren't actually network calls
+# TODO there is duplicated and inconsistent test-python-env setup across this
+#   file, builder/smoke_test.sh, and builder/run_tests.sh, and also in the
+#   conda build scripts themselves. These should really be consolidated
 pkg="/final_pkgs/\$(ls /final_pkgs)"
 if [[ "$PACKAGE_TYPE" == conda ]]; then
   conda install -y "\$pkg" --offline
   retry conda install -yq future numpy protobuf six
+  if [[ "$DESIRED_CUDA" != 'cpu' ]]; then
+    # DESIRED_CUDA is in format cu90 or cu100
```

Comment:
Ew.

Reply:
All the conda installs should be consolidated into a single place anyway. They're spread out across at least three separate files right now